### PR TITLE
Feat/ new staking support

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR adds support for the new (service) staking contract.

* Merged PR #271 which addresses a number of issues with the new staking contracts.
* Fixes issue #280 which caused StakingState be duplicated in different contract packages.

### CODE REVIEW

**Comment by jmoreira-valory** [comment]
This branch should address https://github.com/valory-xyz/trader/issues/280

**Review by dvilelaf — APPROVED** [review]

File: packages/valory/services/trader_omen_gnosis/service.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+name: trader_omen_gnosis
+author: valory
+version: 0.1.0
+description: A set of agents placing bets on omen
```
Comment (Adamantios) [inline packages/valory/services/trader_omen_gnosis/service.yaml:4]:
```suggestion
description: A single agent service (sovereign agent) placing bets on Omen, utilized by Pearl.
```

**Review by Adamantios — COMMENTED** [review]
Looks good. Did you run this with a staked service to verify that it works?

File: packages/valory/skills/staking_abci/rounds.py [hunk 2]
```diff
@@ -42,6 +41,14 @@
...
 
 
+class StakingState(Enum):
+    """Staking state enumeration for the staking."""
+
+    UNSTAKED = 0
+    STAKED = 1
+    EVICTED = 2
+
+
```
Comment (jmoreira-valory) [inline packages/valory/skills/staking_abci/rounds.py:51]: @Adamantios Reverted as discussed

**Comment by jmoreira-valory** [comment]
> Looks good. Did you run this with a staked service to verify that it works?

Yes.

**Review by jmoreira-valory — APPROVED** [review]

**Review by Adamantios — APPROVED** [review]

**Pull request merged**